envoy/http/metadata_interface.h

```diff
@@ -2,10 +2,13 @@
 
 #include <functional>
 #include <iostream>
+#include <sstream>
 #include <memory>
 #include <vector>
 
 #include "absl/container/node_hash_map.h"
+#include "absl/strings/escaping.h"
+#include "fmt/ranges.h"
```

Comment:
nit: should this be changed to "fmt/format.h" or "fmt/stream.h"?

Reply:
change to format.h.